packages/core/src/textures/Texture.js

```diff
@@ -159,10 +159,10 @@ export default class Texture extends EventEmitter
         /**
          * Anchor point that is used as default if sprite is created with this texture.
          * Changing the `defaultAnchor` at a later point of time will not update Sprite's anchor point.
-         * @member {PIXI.Point}
-         * @default {0,0}
+         * @member {?PIXI.Point}
+         * @default null
          */
-        this.defaultAnchor = anchor ? new Point(anchor.x, anchor.y) : new Point(0, 0);
+        this.defaultAnchor = anchor ? new Point(anchor.x, anchor.y) : null;
```

Comment:
Why not just set the second part of the ternary to `setting.SPRITE_DEFAULT_ANCHOR` and call it a day? (maybe `TEXTURE_DEFAULT_ANCHOR` is more appropriate a name too) In general, I think it's better practice to always set defaults than to set to `null` and have to check later.